store/store.css

```
@@ -0,0 +1,125 @@
+@import "reset.css";
+
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  padding: 20px 15px;
+  width: 100vw;
+  font-weight: bold;
+  font-size: 40px;
+  /* block인 경우에는 width 100 설정이 불필요한 것 같긴 한데,, */
+  /*왜 align-items 없이 패딩만 주면 햄버거바가 탈출하는가??*/
+  justify-content: space-between;
+  align-items: center;
+  background-color: lightgreen;
+  color: brown;
+}
+
+body {
+  background-color: lemonchiffon;
+}
+
+main {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav {
+  width: 18vw;
+  height: 25vh;
+  border-radius: 20px; /*radius 설정할 때 왜 px 설정이 %보다 더 자연스러운지 차이 찾아보기*/
+  margin-top: 18px;
+  margin-left: 10px;
+  margin-right: 10px;
```

```suggestion
  margin : 18px 10px 0px 10px
```
이렇게 더 간단하게 적어주면 좋을 듯!!